Lambdas/AutoFaqs/AutoFaqsHandler.swift

```diff
@@ -14,8 +16,12 @@ struct AutoFaqsHandler: LambdaHandler {
     let autoFaqsRepo: S3AutoFaqsRepository
 
     init(context: LambdaInitializationContext) async {
-        self.awsClient = AWSClient()
-        self.autoFaqsRepo = S3AutoFaqsRepository(awsClient: awsClient, logger: context.logger)
+        let httpClient = HTTPClient(
```

Comment:
Do we need to worry about shutting this down?

Reply:
I don't think so considering it'll only need to shutdown when the process is killed.

Reply:
See: https://swift-open-source.slack.com/archives/C5GMX4LP9/p1682667872540799?thread_ts=1682620274.444989&cid=C5GMX4LP9